app/i18n/en-us/feedback.php

```diff
@@ -115,9 +115,9 @@
 		'purge_completed' => 'Purge completed (%d articles deleted)',	// IGNORE
 	),
 	'tag' => array(
-		'created' => 'Tag "%s" has been created.',	// IGNORE
+		'created' => 'Tag “%s as been created.',	// IGNORE
```

Comment:
closing quotation mark is lost

Reply:
🤦

Reply:
Fixed, thanks!

Reply:
A stereotypical French accent. :laughing: